Makefile

```diff
@@ -37,3 +37,27 @@ bblfsh-start:
 		bblfsh/bblfshd\:v2.10.0
 	docker exec style_analyzer_bblfshd bblfshctl driver install \
 		javascript docker://bblfsh/javascript-driver\:v1.2.0
+
+REPORTS_DIR ?= reports
+REPORT_VERSION ?= untagged
+REPORT_DIR ?= $(REPORTS_DIR)/$(REPORT_VERSION)
+SMOKE_REPORT_DIR ?= $(REPORT_DIR)/js_smoke
+NOISY_REPORT_DIR ?= $(REPORT_DIR)/noisy
+QUALITY_REPORT_DIR ?= $(REPORT_DIR)/quality
+SMOKE_INIT ?= ./lookout/style/format/benchmarks/data/js_smoke_init.tar.xz
+
+$(SMOKE_REPORT_DIR) $(NOISY_REPORT_DIR) $(QUALITY_REPORT_DIR):
+	mkdir -p $@
+report-smoke: $(SMOKE_REPORT_DIR)
+	python3 -m lookout.style.format gen-smoke-dataset $(SMOKE_INIT) $(SMOKE_REPORT_DIR)/dataset \
+		2>&1 | tee $(SMOKE_REPORT_DIR)/dataset_logs.txt
+	python3 -m lookout.style.format eval-smoke-dataset $(SMOKE_REPORT_DIR)/dataset \
+		$(SMOKE_REPORT_DIR)/report 2>&1 | tee $(SMOKE_REPORT_DIR)/report_logs.txt
+report-noisy: $(NOISY_REPORT_DIR)
+	python3 -m lookout.style.format quality-report-noisy -o $(NOISY_REPORT_DIR) \
```

Comment:
It should be better not to specify arguments in Makefile but rather use default ones (and change them if required) to avoid having defaults out of sync everywhere

Reply:
there are no defaults and I think like they should not be there.

Reply:
WDYM? There are defaults (https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/cmdline_tools.py#L55 for example)

Reply:
I was not speaking about arguments that are currently mandatory but rather the optional ones that currently have defaults that are overridden here, just to be clear

Reply:
oh, did not see this one you find. I'd prefer just delete this default value.

Reply:
just to always have more explicit way of what exactly we are running

Reply:
it's fine to have default values in cmdline_tools.py, we just need to avoid having another source of truth here

Reply:
this is somewhat orthogonal to this PR though. That some arguments might be better off not having default is one thing, but as long as they do have a default we should not override them here to not have 2 sources of truth. We can create an issue to discuss the CLI but once again this is orthogonal to this PR.

Reply:
I would say that the default values are default exactly because we think that we can provide a sane no-brainer. It makes sense to call the defaults those values which are used to generate the release report. I am totally fine with Hugo's proposal because we print all the configuration in the report. If we don't print some arguments, then we should fix our report.